packages/container/lib/container.js

```diff
@@ -255,7 +270,14 @@ function injectionsFor(container, fullName) {
                                    registry.getTypeInjections(type),
                                    registry.getInjections(fullName));
   injections._debugContainerKey = fullName;
-  injections.container = container;
+
+  setOwner(injections, container.owner);
+
+  // TODO - Inject a `FakeContainer` instead here. The `FakeContainer` will
+  // proxy all properties of the container with deprecations.
+  if (!isEnabled('ember-container-inject-owner')) {
+    injections.container = container;
```

Comment:
It has been in `injectionsFor` to date (see `injections.container = container;` line being removed a few lines above).